src/amo/components/Addon/index.js

```diff
@@ -177,12 +184,24 @@ export class AddonBase extends React.Component {
       isPreviewingTheme,
       installStatus,
     } = this.props;
-    const previewURL = addon ? addon.previewURL : null;
     const type = addon ? addon.type : ADDON_TYPE_EXTENSION;
-    const iconUrl = getAddonIconUrl(addon);
 
-    if (type === ADDON_TYPE_THEME) {
-      const label = isPreviewingTheme ? i18n.gettext('Cancel preview') : i18n.gettext('Tap to preview');
+    // The 'tap to preview' button will be fully removed in the following PR:
+    // https://github.com/mozilla/addons-frontend/pull/4914
+    // for now this is only displayed over lightweight preview images
```

Comment:
I guess you dont need this comment here.

Reply:
I thought it was a good idea to leave in till the PR gets merged (it's currently in a "do not merge" state). I am just curious why you think i don't need it?

Reply:
ok i see i guess i was just explaining that this was somewhat temporary and reasons for some differences in handling of static vs lightweight themes.

Reply:
I guess I can remove it